tests/check_array.c

```diff
@@ -143,23 +140,23 @@ START_TEST(test_take1_ok)
   dims[1] = 2;
   dims[2] = 3;
   ga_assert_ok(GpuArray_reshape_inplace(&vres, 3, dims, GA_C_ORDER));
-  
+
   ga_assert_ok(GpuArray_take1(&vres, &v, &vidx, 0));
   ga_assert_ok(GpuArray_read(buf, sizeof(uint32_t) * 72, &vres));
 
   /* 0 */
-  ck_assert(buf[ 0] == 18);
-  ck_assert(buf[ 1] == 19);
-  ck_assert(buf[ 2] == 20);
-  ck_assert(buf[ 3] == 21);
-  ck_assert(buf[ 4] == 22);
-  ck_assert(buf[ 5] == 23);
+  ck_assert(buf[0] == 18);
+  ck_assert(buf[1] == 19);
+  ck_assert(buf[2] == 20);
+  ck_assert(buf[3] == 21);
+  ck_assert(buf[4] == 22);
+  ck_assert(buf[5] == 23);
 
   /* 1 */
-  ck_assert(buf[ 6] == 18);
-  ck_assert(buf[ 7] == 19);
-  ck_assert(buf[ 8] == 20);
-  ck_assert(buf[ 9] == 21);
+  ck_assert(buf[6] == 18);
+  ck_assert(buf[7] == 19);
+  ck_assert(buf[8] == 20);
+  ck_assert(buf[9] == 21);
```

Comment:
It's not a huge deal.  If your tool can fix most other things but this, it would be ok.